src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,41 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static getAlive(animal) {
+    Animal.alive = Animal.alive.filter(beast => beast !== animal);
+  };
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
   // write your code here
+  bite(animal) {
+    if (animal.hidden === false) {
+      animal.health -= 50;
+    }
```

Comment:
you should also check the type of animal here: is it Harbivore or Carnivore
check the task description

Reply:
Ok